packages/bot/src/stores/summary-store.js

```diff
@@ -16,12 +15,14 @@ export default class SummaryStore {
         this.root_store        = root_store;
         const { client }       = this.root_store.core;
         this.currency          = client.currency;
-
-        observer.register('contract.status', this.onContractStatusEvent);
     }
 
     @action.bound
     onContractStatusEvent(contract_status) {
+        if (!this.root_store.core.client.is_virtual) {
+            // TODO remove after activate real account
+            return;
+        }
```

Comment:
What happens if this is removed? Since we can't trade on real accounts, will the logic below be executed?

Reply:
@aaron-binary also here, since we are not letting user to trader on virtual account i don't think we need to check here.